### PR TITLE
Robust scrape

### PR DESCRIPTION
**Pull request by MinhPhu0304**
Add Sentry alerting

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,13 +69,30 @@ async function getLocations(lat, lng, cursor) {
...
+    throw new Error("Stop scraper")
+  }
   const data = await res.json()
   return data
 }
 
 async function main () {
 
   const pointsToCheck = await getAllPointsToCheck()
+  if (pointsToCheck.features.length === 0) {
```
Comment (MinhPhu0304) [inline index.js]: Prevent issue when MoH endpoint data is down.

File: index.js [hunk 2]
```diff
@@ -63,13 +69,30 @@ async function getLocations(lat, lng, cursor) {
...
+      scope.addBreadcrumb({
+        data: res.status,
+        message: res.text()
+      })
+      scope.addBreadcrumb({
+        data: res.text(),
+        message: 'Response string'
+      })
+    });
+    throw new Error("Stop scraper")
```
Comment (MinhPhu0304) [inline index.js]: Throw error from here then let it propagate all the way to main function then exit to stop committing bad data

**Comment by noway** [comment]
Good stuff 👍🏻 
Will give a proper review later

**Review by noway — APPROVED** [review]
LGTM

**Review by wseagar — APPROVED** [review]

**Pull request merged**